### ISSUE TITLE
npm_config_prefix default shouldn't be a relative path

### ISSUE THREAD

**Issue by louim** [geerlingguy/ansible-role-nodejs#29]
Hello!

#28  added a default `npm_config_prefix` of `~/.npm-global`, and this path is added to the global path (for all users). So for all other user except the `ansible_ssh_user`, this path will not resolve because it will be expanded as their home folder.

I think a better default would be something like:  `/usr/local/lib/npm`, so the path will resolve for all users.

What do you think?
cc @thom8

**Comment by thom8** [comment]
@louim yeah I agree that's a saner default for this role.

I was using the context of a VM provisioned with a single non admin user whereas this role can be used elsewhere in a multi user environment.

Just need to override this variable in the default config of [Drupal VM](https://github.com/geerlingguy/drupal-vm/blob/master/default.config.yml) to avoid introducing a regression https://github.com/geerlingguy/drupal-vm/issues/629

**Comment by sarahjean** [comment]
I was just trying to get something similar working, with the latest version of Drupal VM. Here's the config I was trying to use in the config.yml:

```
# `nodejs` must be in installed_extras for this to work. Valid examples: "0.10",
# "0.12", "4.x", "5.x".
nodejs_version: "4.x"

nodejs_npm_global_packages: 
  - name: gulp
  - name: browser-sync
  - name: avn
  - name: avn-nvm

nodejs_install_npm_user: "vagrant"
npm_config_prefix: "~/.npm-global"
npm_config_unsafe_perm: "true"
```

I have also tried it without that last variable, and with the alternate option of `npm_config_prefix: "/usr/local/lib/npm"`

The install for global packages just keeps failing. I will try to attach npm-debug.log as well, if that will be of any use.
[npm-debug.log.txt](https://github.com/geerlingguy/ansible-role-nodejs/files/307628/npm-debug.log.txt)

**Comment by louim** [comment]
@sarahjean are you sure you are running the latest version of the role? The log make it look like it is trying to write to `/usr/lib/node_modules/avn-nvm`.

**Comment by sarahjean** [comment]
Not the latest, do I need to be on 3.1.0 for this? The latest tagged version of drupal-vm is running this
```
- src: geerlingguy.nodejs
  version: 3.0.0
```

**Comment by louim** [comment]
Yep: https://github.com/geerlingguy/ansible-role-nodejs/compare/3.0.0...3.1.0

**Comment by geerlingguy** [comment]
Fixed via #31

**Comment by geerlingguy** [comment]
This will be in `3.2.0` (and I'll have Drupal VM use that role version for it's 3.1.1 release).
